### PR TITLE
event actions api and guide

### PR DESCRIPTION
**Pull request by fxleblanc**
# Event actions documentation

This PR aims to clarify the usage of the upcoming event actions endpoint.

## Event actions API

event-actions-new-design

### CODE REVIEW

File: docs/guides/ignore-events.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (aviau) [inline docs/guides/ignore-events.mdx:1]: I would remove this page. I think the endpoint is simple enough that the endpoint page is enough. This does not add much.

File: docs/introduction/getting-started.mdx [hunk 2]
```diff
@@ -58,6 +58,10 @@ The Flare API can be used to access Flare's search capabilities, configure monit
     Monitor leaked cookies for your domain.
   </Card>
 
+  <Card title="Ignore/Remediate Events" icon="check" href="/guides/ignore-events">
```
Comment (aviau) [inline docs/introduction/getting-started.mdx:61]: Remove this.

File: docs/api-reference/v4/endpoints/event-actions.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Apply Action to Events"
+api: "POST https://api.flare.io/firework/v4/events/actions/{action}"
```
Comment (sebd11) [inline docs/api-reference/v4/endpoints/event-actions.mdx:3]: I thought we said the action would be in the payload?

Comment (fxleblanc) [reply docs/api-reference/v4/endpoints/event-actions.mdx:3]: Discussion is ongoing in the [RFC](https://flareio.atlassian.net/wiki/spaces/engineering/pages/609222671/2024-12-17+-+Remediate+and+ignore+activities+in+the+Public+API). Will update the PR when the final solution is agreed upon.

**Comment by fxleblanc** [comment]
Ready for review.

File: docs/api-reference/v4/endpoints/event-actions.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+Apply an action to a list of events(ignore, remediate, etc...).
+
+## Path parameters
+
+<ParamField body="action" type="object" required>
+  The action applied to events.
+
+
```
Comment (aviau) [inline docs/api-reference/v4/endpoints/event-actions.mdx:13]: no need two empty lines here?

**Comment by aviau** [comment]
lgtm but we don't merge right now.

**Pull request merged**